src/workerd/api/unsafe.h

```diff
@@ -49,14 +49,22 @@ class UnsafeEval: public jsg::Object {
       jsg::Arguments<jsg::JsRef<jsg::JsString>> args,
       const jsg::TypeHandler<UnsafeEvalFunction>& handler);
 
+  // We could expand this to allow not just Uint8Array but a general BufferSource
+  jsg::JsValue newWasmModule(jsg::Lock& js, v8::Local<v8::Uint8Array> src);
```

Comment:
You can pass the `src` in here as a `kj::Array<kj::byte>`. The type wrapper system will automatically handle the conversion. It'll save you a few lines in your impl.

Reply:
Presumably that also makes it work for a general `BufferSource`?